### ISSUE TITLE
Function visibility issues and incorrect function name in examples/jsonClient.go

### ISSUE THREAD

**Issue by pradykaushik** [paypal/gorealis#46]
In [jsonClient.go](https://github.com/paypal/gorealis/blob/master/examples/jsonClient.go#L92), `realis.NewDefaultConfig(...)` shows an **Unresolved reference** error.
Assuming that [this](https://github.com/paypal/gorealis/blob/master/realis.go#L379) is the function that needs to be called, then we will need to capitalise the name from  _newDefaultConfig_ to _NewDefaultConfig_ for inter-package visibility.

Also, in [jsonClient.go#99](https://github.com/paypal/gorealis/blob/master/examples/jsonClient.go#L99) the call to `realis.NewClient(...)` shows an error as there is no function named _NewClient_ in the _realis_ package.
I looked through older versions of the code and found the function [here](https://github.com/paypal/gorealis/blob/d7efa913b13ff9b875a8038270660977c9231c2d/realis.go#L60).
Should one use `NewRealisConfig(option ...ClientOption)` instead?

**Comment by rdelval** [comment]
The json client example hasn't been updated in a long time and the API it depends on was deprecated and remove. It could definitely use an update to be brought back into sync with the library.

**Comment by pradykaushik** [comment]
For the JSON client, there are three possibilities,
1. Take in three JSON files from the command line,
    -- [job.json](https://github.com/paypal/gorealis/blob/master/examples/job.json), [clusters.json](https://github.com/paypal/gorealis/blob/master/examples/clusters.json), and a `config.json` (specifies information such as _username_, _password_, _schedulerURL_, _zkURL_, etc.)

2. Take in only two JSON files, [job.json](https://github.com/paypal/gorealis/blob/master/examples/job.json), and [clusters.json](https://github.com/paypal/gorealis/blob/master/examples/clusters.json). Configuration information such as _username_, _password_, _schedulerURL_, _zkURL_ etc. will be passed directly via command line and not in the form of a JSON file.

3. Embed [clusters.json](https://github.com/paypal/gorealis/blob/master/examples/clusters.json) into `config.json`. Then `config.json` be unmarshalled into [Cluster](https://github.com/paypal/gorealis/blob/master/clusters.go#L24) and the following _Config_ struct.
```go
type Config struct {
    username string
    password string
    schedulerURL string
    zkURL string
}
```

@rdelval I would like hear your thoughts on this.

**Comment by rdelval** [comment]
Embedding a minimal subset of `clusters.json` into `config.json` sounds like the best option here.

**Comment by pradykaushik** [comment]
Resolved -- #67 for patch.
